tools/rpmbuild.cc

```diff
@@ -160,6 +166,9 @@ static struct poptOption rpmBuildPoptTable[] = {
  { "bi", 0, POPT_ARGFLAG_ONEDASH, 0, POPT_BI,
 	N_("build through %install (%prep, %conf, %build, then install) from <specfile>"),
 	N_("<specfile>") },
+ { "bk", 0, POPT_ARGFLAG_ONEDASH, 0, POPT_BK,
+	N_("build through %install (%prep, %conf, %build, %install, then check) from <specfile>"),
```

Comment:
Shouldn't this (and the other variants below) say "build through %check"?

Reply:
Yep, dangers of copy-paste :smile: